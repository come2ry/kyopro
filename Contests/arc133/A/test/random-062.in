10
2 3 3 11 7 10 3 3 6 7
