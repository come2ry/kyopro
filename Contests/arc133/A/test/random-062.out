2 1 1 7 1 0 6 7
